Riot/Modules/VoiceBroadcast/VoiceBroadcastSDK/VoiceBroadcastAggregator.swift

```diff
@@ -168,7 +170,10 @@ public class VoiceBroadcastAggregator {
                   let state = VoiceBroadcastInfoState(rawValue: voiceBroadcastInfo.state) else {
                 return
             }
-
+            // For .pause and .stopped, if there is a lastChunkSequence (ie its value is > 0), we store it
+            if [.stopped, .paused].contains(state), voiceBroadcastInfo.lastChunkSequence > 0 {
+                self.voiceBroadcastLastChunkSequence = voiceBroadcastInfo.lastChunkSequence
```

Comment:
I think we should update the stored value `self.voiceBroadcastLastChunkSequence` on each .paused or .stopped event without checking the value, by considering 0 as the default value if none.

Indeed if the .stopped mention 0, this is  better to take it into account and reset a potential value stored from a previous .paused. We sent 0 with .stopped when we stop an unexpected running VB

Reply:
@giomfo I updated the code accordingly.